### PR TITLE
Statements encapsulate

### PR DESCRIPTION
**Pull request by joelmdesouza**
@crgimenes @felipeweb 
prest/prest#275

### CODE REVIEW

File: postgres/statements/permissions.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package statements
+
+const (
+	READ   string = "read"
+	WRITE  string = "write"
+	DELETE string = "delete"
+)
```
Comment (felipeweb) [inline postgres/statements/permissions.go]: All letters don’t need to be capitalized just the first letter

**Review by felipeweb — CHANGES_REQUESTED** [review]
All these functions needs to be used in controllers. Where is the controllers’s pull request?

File: postgres/statements/permissions.go (hunk 1 above)
Comment (joelmdesouza) [reply postgres/statements/permissions.go]: removed permisions.go
is not used on adapters

**Comment by joelmdesouza** [comment]
Sent the controllers PR @felipeweb

**Review by marioidival — APPROVED** [review]

**Review by crgimenes — APPROVED** [review]
LGTM

**Pull request merged**